src/integrations/GA4/ECommerceEventConfig.js

```diff
@@ -169,7 +169,7 @@ const eventNamesConfigArray = [
     hasItem: true,
     includeList: [
       eventParametersConfigArray.Currency,
-      eventParametersConfigArray.Total,
+      eventParametersConfigArray.Price,
```

Comment:
It looks like a breaking change. Can't we support both the total and price?
```suggestion
      eventParametersConfigArray.Total,
      eventParametersConfigArray.Price,
```

Reply:
We could, but that essentially means we are deviating from the rudderstack ecommerce spec.
For example, in this event (Product Viewed), we do not support a total property in the root ([reference](https://www.rudderstack.com/docs/event-spec/ecommerce-events-spec/ordering/#product-viewed)) and rather support a `price` property. 
We could though keep it for backward compatibility purposes, if this is okay.

Reply:
Let's keep it backword compatible.

Reply:
Done